general.css

```diff
@@ -0,0 +1,137 @@
+*,
+::after,
+::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+:root {
+  --color-black: black;
+}
+::-webkit-scrollbar{
+    background-color: aliceblue;
+    width:8px;
+}
+::-webkit-scrollbar-thumb{
+    border-radius: 1000px;
+    background-color: rgb(83, 79, 79);
+    border:2px solid #111;
+}
+::-webkit-scrollbar-thumb:hover{
+    background-color: rgb(170, 177, 177);
+}
+html{
+  height:100%;
+}
+body {
+  font-family: 'Nunito Sans', sans-serif;
+  display:flex;
+  flex-direction:column;
+  height:100%;
+
+}
+a {
+  text-decoration: none;
+}
+header {
+  --color-white: white;
+  --color-header-route:#f3f2ee;
+}
+
+.header-top {
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--color-black);
+  color: var(--color-white);
+  padding: 15px;
+  padding-left: 100px;
+  padding-right: 100px;
+}
+.header-top a {
+  color: var(--color-white);
+}
+.header-top nav {
+  display: flex;
+  gap: 20px;
+}
+.header-bottom {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  padding-left: 100px;
+  padding-right: 100px;
+  background-color: var(--color-white);
+  align-items: center;
+}
```

Comment:
header-top and bottom are same, so try re-using the lines.

Reply:
I have updated it Thank you for your suggestions. Can you please approve it